core/kazoo_transactions/src/wht_util.erl

```diff
@@ -272,13 +274,13 @@ current_account_dollars(Account) ->
 %%
 %% @end
 %%--------------------------------------------------------------------
--spec get_balance_from_account(ne_binary(), couch_util:view_options()) -> units().
+-spec get_balance_from_account(ne_binary(), kazoo_modb:view_options()) -> units().
 get_balance_from_account(Account, ViewOptions0) ->
     View = <<"transactions/credit_remaining">>,
     AccountId = kz_util:format_account_id(Account),
     AccountDb = kz_util:format_account_db(AccountId),
     ViewOptions = ['first_when_multiple' | ViewOptions0],
-    case kz_datamgr:get_results(AccountDb, View, ViewOptions) of
+    case kz_datamgr:get_results(AccountDb, View, kazoo_modb:strip_modb_options(ViewOptions)) of
```

Comment:
but `first_when_multiple` won't get processed in `kz_datamgr:get_results` so maybe we need to call `kz_datamgr:get_single_result`.